Contents/MacOS/startupRAMDiskandCacheMover.sh

```diff
@@ -201,6 +201,54 @@ move_chrome_chanary_cache()
    fi
 }
 
+#
+# Brave Cache
+#
+
+# Create a directory for Brave in the ramdisk
+make_brave_dir()
+{
+    /bin/mkdir -pv "${USERRAMDISK}"/Brave-Browser
+}
+
+# Move existing Brave cache to ramdisk
+move_brave_cache()
+{
+    /bin/mkdir -p /tmp/Brave-Browser
+    /bin/mv ~/Library/Caches/BraveSoftware/Brave-Browser/* /tmp/Brave-Browser
+    /bin/mv /tmp/Brave-Browser/* "${USERRAMDISK}"/Brave-Browser
+    /bin/rm -rf ~/Library/Caches/BraveSoftware/Brave-Browser
+    
+}
+
+# Link Brave cache to ramdisk
+link_brave_dir()
+{
+    /bin/ln -v -s "${USERRAMDISK}"/Brave-Browser ~/Library/Caches/BraveSoftware/Brave-Browser
+}
+
+# Check what to do with brave
+check_brave_cache()
```

Comment:
Thank you for the PR.
What if you simplify adding this browser by joining all the separate functions you did into one as it is done for other browsers, i.e chrome?

The `ln` behaviour should not be changing, it will break many things for them I suspect.

Reply:
I broke it up into several functions because I needed to do several more than once to prepare for previously-enabled cache. If you want me to do it I can make it a single function and... copy and paste some code

Reply:
And speaking of the `ln` behaviour: it cost me a lot of time to figure out the problem. I was not expecting it as system tools usually behave as expected